### ISSUE TITLE
Podpora azijskych znakov

### ISSUE THREAD

**Issue by Branicek** [marsher/volby-zo-zahranicia#164]
Zdravim, chcel som si vytvorit ziadost a poslat do ciny ale cinske znaky sa vo vytvorenej ziadosti nezobrazuju

**Comment by scholtz** [comment]
no neviem ci by to malo byt podporovane.. neviem ako budu uradnicky z tej ziadosti potom obkreslovat tie znaky :) Neexistuje nejaky zapis latinkou pre cinske adresy?

stranka pouziva utf-8, a vyzera to tak, ze funcionalita kniznice [PDFMake](https://github.com/bpampuch/pdfmake) je v tomto asi obmedzujuci faktor..

**Comment by Branicek** [comment]
mohli by to vytlacit, ako ked si objednavam nieco postou z krajiny mimo ciny. pretoze zapis latinskymi znakmi cinania nepochopia, je tam prilis vela "ambiguity". uz som to skusal raz ked som este nevedel pisat cinsky a neprislo mi nic :D nevadi, skusim to nejak manualne poslat a presvedcit nech to vytlacia a nalepia alebo co

**Comment by Kepro** [comment]
myslim ze treba do PDFMake pridat font ktory ma cinske znaky.. my sme tak riesili jeden krat sme zohnali Arial font ktory mal 20MB :D 

btw vsetko z chiny co mi prislo bola spatna adresa pisana po anglicky aj adresy aj mena cinanov

**Comment by peterppp** [comment]
Tento system vygeneruje hotove PDFko ako prilohu do e-mailu. Neprenasa udaje do systemu obecneho uradu.
Takze uradnici by to asi museli rucne prekreslovat, pripadne Ctrl+C a Ctrl+P - ale to im bude fungovat iba ak budu mat nainstalovane Cinske fonty na ich pocitaci. Pravdepodobnost je velmi mala.

**Comment by Kepro** [comment]
tak potom je to nepodstatne, na urade to vyhodia rovno...

**Comment by gamelaster** [comment]
Možno by to šlo urobiť cestou napr. z Japončiny urobiť Romaji (teda v našej abecede) preklad.
Určite sa takýto preklad dá urobiť aj z Čínštiny

**Comment by Kepro** [comment]
@GAMELASTER  bral by si na seba to riziko spravneho prekladu z cinstiny? :)

**Comment by gamelaster** [comment]
@Kepro No, to asi nie :-D Ako, troška som to viac skúmal, a je to oveľa väčší problém ako napr. u Japončiny, ale i tak, myslím si že mať podporu pre čínske a japonské znaky je zbytočné, určite u obydvoch jazykov ide napísať adresu v "normálnej" abecede

**Comment by jhedgehog** [comment]
Zbytocne to urcite nie je. Mne prisiel z uradu email, ze sa im list vratil. Odpisal som im aby my list zaslali opat a zaslal som im moju adresu v Simplified Chinese. 

Do buducna navruhujem, aby sa pouzitvatelom stranky ktory zvolia v drop down menu Cinu, Taiwan, Japonsko a dalsie krajiny ktore nepouzivaju latinku, zobrazila informacia typu: "Pre istotu, zaslite dodatocny email na Vas urad (vygenerovany email) s adresou zapisanou v Simplified Chinese (Traditional Chinese, etc.)"
